src/app/checkout/step-2/credit-card/credit-card.tpl.html

```diff
@@ -1,124 +1,153 @@
-<div class="mb">
+<form novalidate ng-submit="$ctrl.submitPayment(creditCardPayment.$valid)" name="$ctrl.creditCardPaymentForm">
```

Comment:
I'm not using `on-submit` since the submit button is outside the form and component. It just got left there. I'll remove it from both the credit card and bank account templates.